locodedb/calls.go

```diff
@@ -0,0 +1,94 @@
+package locodedb
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+var errRecordNotFound = errors.New("record not found")
+
+type LOCODE [2]string
+
+var (
+	mCountries map[CountryCode]string
+
+	mLocodes map[string]locodesCSV
+)
+
+// Get returns a record for a given LOCODE string.
+func Get(locodeStr string) (rec Record, err error) {
+	if err := initLocodeData(); err != nil {
+		return rec, err
+	}
+
+	var newLocode *LOCODE
+	newLocode, err = FromString(locodeStr)
+	if err != nil {
+		return rec, err
+	}
+
+	key, err := NewKey(*newLocode)
+	if err != nil {
+		return rec, err
+	}
+	return GetFromKey(*key)
+}
+
+// GetFromKey returns a record for a given LOCODE key, which is Key.
+func GetFromKey(key Key) (rec Record, err error) {
+	rec = Record{}
```

Comment:
you should either not init it like this (named return values do the work for you) or drop the named return values (the preferred way to me, but lets also init var as `var rec Record`)

Reply:
names are dropped